cmd/cone/flags.go

```diff
@@ -68,3 +69,7 @@ func addForceTaskCreateFlag(cmd *cobra.Command) {
 func addEntitlementDetailsFlag(cmd *cobra.Command) {
 	cmd.Flags().Bool(extraDetailsFlag, false, "Show more details about the app and entitlement for this request")
 }
+
+func addRawTokenFlag(cmd *cobra.Command) {
+	cmd.Flags().Bool(rawTokenFlag, false, "Prints only the access token")
```

Comment:
Change the desc to "Prints only the access token directly to stdout without style"